### ISSUE TITLE
Homebrew problem

### ISSUE THREAD

**Issue by Trimint123** [SourMesen/Mesen#386]
So I was trying this Touhou Homebrew. But looks like it got a background problem.
![touhourououmu_000](https://user-images.githubusercontent.com/25479107/40399228-605fccb0-5e66-11e8-8802-991d8bf32a7d.png)
Sprite is okay, but the background is shake with lines all over the place. I decide to test it with other emulator, and it seems to work fine.

For testing purpose, I will share that game here:
http://kyoske.sakura.ne.jp/index.cgi?p=%E6%9D%B1%E6%96%B9%E8%80%81%E6%A1%9C%E5%A4%A2

And for an extra note, that shake suddenly gone if you set the region to PAL.

**Comment by retro-wertz** [comment]
that game is probably a PAL mode game then,,,

**Comment by Trimint123** [comment]
No, other emulator I tested are in NTSC and it works fine as expected.

**Comment by SourMesen** [comment]
Thanks for the report.  Assuming the game works properly on hardware (I have no way of testing this, but I'm going with the assumption it might), this looks like another case of being off by a single PPU cycle, which ends up causing scrolling glitches.

Fixing this will probably take a while - changes to these sort of things inevitably will break other scenarios and it usually takes a while to find a solution that works for all scenarios at once (Bizhawk's code might be a good lead in this case since it plays the game correctly, and passes more or less the same tests as Mesen does)

**Comment by Trimint123** [comment]
Thanks for information. Hopefully those can be solve soon.

**Comment by Trimint123** [comment]
Oh, I forgot to mention. It's not just the title screen, but also in-gameplay as well. But well, that could be already being fixed. Thanks!

**Comment by SourMesen** [comment]
I played some of the game itself too, it seems to be working fine.  If you're using Windows, you can try this dev build which contains the fix: https://www.mesen.ca/MesenJune5.zip

**Comment by Trimint123** [comment]
Again, thanks for fixing it.    Pada Rabu, 6 Juni 2018 09.30.10 WIB, SourMesen <notifications@github.com> menulis:  
 
 
I played some of the game itself too, it seems to be working fine. If you're using Windows, you can try this dev build which contains the fix: https://www.mesen.ca/MesenJune5.zip
